src/layout/panel/workspaceList.js

```diff
@@ -553,22 +561,53 @@ var WorkspaceButton = GObject.registerClass(
         }
 
         handleDragOver(source, actor, x, y) {
-            if (!(source instanceof WorkspaceButton) || !this.draggable) {
```

Comment:
I would prefer that we keep testing on class instance like this but also with the `TaskButton` class (which need to be imported)